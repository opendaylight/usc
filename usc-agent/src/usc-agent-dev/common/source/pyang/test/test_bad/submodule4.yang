submodule submodule4 {

  belongs-to composite {
    prefix c; 
  }
  include submodule2;
}
