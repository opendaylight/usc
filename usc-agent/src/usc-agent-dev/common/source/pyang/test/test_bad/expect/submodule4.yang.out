submodule2.yang:7: error: BAD_IMPORT
submodule2.yang:7: warning: UNUSED_IMPORT
